modules/era/sql/gardening_results_era.sql

```diff
@@ -2,26 +2,25 @@
 -- Gardening Module to remove Results based on Era by Expansion
 ---------------------------------------------------------------
 
+UPDATE gardening_results
+SET weight = 0 WHERE resultid IN 
+(
 -- Chains of Promathia
--- DELETE FROM gardening_results WHERE resultId = 34;  -- Eggplant
--- DELETE FROM gardening_results WHERE resultId = 647; -- Sprig Of Misareaux Parsley
--- DELETE FROM gardening_results WHERE resultId = 664; -- Holy Basil
--- DELETE FROM gardening_results WHERE resultId = 667; -- Vanilla
-
--- Treasures of Aht Urghan
--- DELETE FROM gardening_results WHERE resultId = 2423; -- Gregarious Worm
--- DELETE FROM gardening_results WHERE resultId = 2426; -- Clump of Garidav Wildgrass
--- DELETE FROM gardening_results WHERE resultId = 2428; -- Zegham Carrot
--- DELETE FROM gardening_results WHERE resultId = 2429; -- Azouph Greens
--- DELETE FROM gardening_results WHERE resultId = 2434; -- Vomp Carrot
--- DELETE FROM gardening_results WHERE resultId = 2437; -- Sharug Greens
--- DELETE FROM gardening_results WHERE resultId = 2440; -- Clump Of Tokopekko Wildgrass
--- DELETE FROM gardening_results WHERE resultId = 2442; -- Cupid Worm
--- DELETE FROM gardening_results WHERE resultId = 2444; -- Cupid Worm
-
+--34,  -- Eggplant
+--647, -- Sprig Of Misareaux Parsley
+--664, -- Holy Basil
+--667, -- Vanilla 
+--2423, -- Gregarious Worm
+--2426, -- Clump of Garidav Wildgrass
```

Comment:
Did you want these to still be commented out? currently the only thing that will actually be not gardened is the winterflower.

Reply:
In TR's we don't have anything commented out, but I figured in ASB you guys would want them since they are in 75 era still technically.

Reply:
Can you organizes the commented out ones by 'era' then and label them?